glossary.md

```diff
@@ -141,6 +245,36 @@ PD 中的 Store 指的是集群中的存储节点，也就是 tikv-server 实例
 
 ## T
 
+### Temporary table
+
+临时表 (temporary table) 解决了业务中间计算结果的临时存储问题，让用户免于频繁地建表和删表等操作。用户可将业务上的中间计算数据存入临时表，用完数据后 TiDB 自动清理回收临时表。这避免了用户业务过于复杂，减少了表管理开销，并提升了性能。
+
+详情参见[临时表](/temporary-tables.md)。
+
+### TiDB Data Migration (DM)
```

Comment:
Maybe there should also be a direct entry for "DM" like "DM: See 'TiDB Data Migration'"